osi_environment.proto

```diff
@@ -119,18 +128,31 @@ message EnvironmentalConditions
         // Other (unspecified but known) ambient illumination.
         //
         AMBIENT_ILLUMINATION_OTHER = 1;
-        // Ambient illumination index is "bright".
-        // E.g. daytime with sunshine.
-        AMBIENT_ILLUMINATION_BRIGHT = 2;
-        // Ambient illumination index is "medium".
-        // E.g. daytime with overcast sky.
-        AMBIENT_ILLUMINATION_MEDIUM = 3;
-        // Ambient illumination index is "low".
-        // E.g. during twilight (dawn/dusk) or in a tunnel.
-        AMBIENT_ILLUMINATION_LOW = 4;
-        // Ambient illumination index is "dark".
-        // E.g. night time, no sunlight.
-        AMBIENT_ILLUMINATION_DARK = 5;
+        // Illumination between [0.001, 0.01[ Lux is Level 1.
+        // E.g. Night with no artificial light.
+        AMBIENT_ILLUMINATION_LEVEL1 = 2;
```

Comment:
I am not sure about the naming here:
The meaning of LevelXY is somehow not easy to grasp... but I understand that it is hard to find a more suitable naming convention...